backend/manager/modules/restapi/jaxrs/src/main/java/org/ovirt/engine/api/restapi/resource/BackendDiskSnapshotsResource.java

```diff
@@ -42,7 +42,8 @@ protected DiskSnapshots mapCollection(List<org.ovirt.engine.core.common.business
             collection.getDiskSnapshots().add(addLinks(populate(diskSnapshot, disk), Disk.class));
             diskSnapshot.setHref(buildHref(diskId.toString(), diskSnapshot.getId().toString()));
             if (diskSnapshot.getParent() != null) {
-                diskSnapshot.getParent().setHref(buildParentHref(diskId.toString(), false));
```

Comment:
@bennyz I see `buildParentHref()` was used only twice (now only once) and always with `addDiskSnapshots` parameter equal to `false`.
Maybe we don't need it at all?
Can use `buildHref()` instead, just pass `null` for `snapshotId` and test for `disksnapshots` not being `null` before adding `"/disksnapshots/" + snapshotId` suffux?